frontend/src/metabase/components/EmojiIcon.jsx

```diff
@@ -0,0 +1,18 @@
+/* eslint "react/prop-types": "warn" */
```

Comment:
Yep that's the plan.

Although for a few of the components like Item and List I ended up making some changes that aren't compatible with the questions implementation yet. I'll try to get those refactored after finishing the rest of the feature.